spec/ameba/rule/style/verbose_block_spec.cr

```diff
@@ -89,9 +89,11 @@ module Ameba::Rule::Style
         rule = VerboseBlock.new
         rule
           .tap(&.exclude_prefix_operators = true)
+          .tap(&.exclude_operators = true)
```

Comment:
Only the test on line 94 actually needed an accompanying `exclude_operators` setting to pass, but I've added it to both tests for consistency until we figure out if this is an acceptable solution.

Reply:
Setting it here is not needed.

Reply:
Right, I was mainly setting it in case we change the defaults again, to make it more apparent that the prefix setting is closely tied to the more general operator setting.

I can remove it!

Reply:
Please do as it's not needed :)